Client/src/Components/CheckboxTree/CheckboxTree.tsx

```diff
@@ -790,8 +796,9 @@ export default class CheckboxTree<T> extends Component<Props<T>, State<T>> {
           shouldExpandOnClick={shouldExpandOnClick}
           getNodeId={getNodeId}
           getNodeChildren={getStatefulChildren}
-          renderNode={this.renderNode} />
-      )}
+          renderNode={this.renderNode}
+          customCheckboxes={customCheckboxes as unknown as CustomCheckboxes<StatefulNode<T>>} />
```

Comment:
I don't see a direct way to remove this casting. It might require bigger changes that are well beyond the scope of this unit of work.

Reply:
Yeah, it's pretty nasty!